.github/workflows/publish.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: Upload SPDX Python Bindings
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  test:
+    uses: ./.github/workflows/test.yaml
+
+  deploy:
+    needs:
+      - test
+
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.x'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install build
+
+      - name: Build package
+        run: python -m build
+
```

Comment:
Question: until we have a PyPI repo, do we want to publish artifacts that can be installed from Git or some other repo?  I'm not sure exactly how to do this - perhaps something similar to how we install the license matcher [in this requirements file](https://github.com/spdx/spdx-online-tools/blob/f4aa1ca28f10d811567cd543d960768714665cb1/requirements.txt#L20).

Reply:
It's possible to do this without any changes. I'll add instructions on how to do this to the README